tests/app/celery/test_tasks.py

```diff
@@ -2121,8 +2148,39 @@ def test_send_inbound_sms_to_service_does_not_retries_if_request_returns_404(not
     mocked.call_count == 0
 
 
-def test_process_incomplete_job_sms(mocker, sample_template):
+def test_process_incomplete_job_sms_FF_PRIORITY_LANES_true(mocker, sample_template):
```

Comment:
can we move this below the test at line # 483 and make them both a class

Reply:
it's not really that those tests should be together in a class, but rather that all the `process_jobs()` tests should. Let me spend a couple minutes reorging this file...